unit_tests/test_IncrementalSVD.cpp

```diff
@@ -138,6 +139,42 @@ TEST(IncrementalSVDSerialTest, Test_getSingularValues)
     }
 }
 
+TEST(IncrementalSVDSerialTest, Test_FastUpdate)
```

Comment:
This file is for the test of the base class. It may be better to move this inherited class test into a separate test file.